init-environment.sh

```diff
@@ -0,0 +1,25 @@
+# Initializes environment variables necessary for running
+# tests against docker containers.
+
+alias echoerr=">&2 echo"
+
+# set up trap so that script can be exited from within functions
+trap "exit 1" TERM
+main_pid=$$
+
+lookup_service_port() {
+    service=$1
+    port=$2
+    port_mapping_output=$(docker-compose -f docker-compose.yml port $service $port)
+    if [ $? -ne 0 ]; then
+        echoerr "$service container not running. Exiting early."
+        kill -s TERM $main_pid
```

Comment:
I wanted to force this to exit the script early if we fail to find the containers running. A simple `exit` will only exit the function, not the entire script. Have to declare a trap above, send a TERM signal to the process running this script, and have that trap execute the exit.